doc/configure-ssl.md

```diff
@@ -13,4 +13,4 @@ sudo service nginx configtest
 sudo service nginx reload
 ```
 
-Then you can add the AlchemyRootCA (located at `~/ssl/AlchemyRootCA.pem`) in your favorite browser. 
+Then you can add the AlchemyRootCA (located at `~/ssl/AlchemyRootCA.p12`) in your favorite browser. 
```

Comment:
`.pem` is working for me in Chrome, why do you need it?

Reply:
Version 88.0.4324.150 (Official Build) (64-bit)

pem don't show up if i don't select "all files", and error when imported.
![image](https://user-images.githubusercontent.com/1247618/108194767-48882300-7117-11eb-9ee0-f5b79498834c.png)
![image](https://user-images.githubusercontent.com/1247618/108194877-6b1a3c00-7117-11eb-8bcc-7bf410ab452c.png)
![image](https://user-images.githubusercontent.com/1247618/108194528-09f26880-7117-11eb-94bc-4ed9d40b1adc.png)

p12 is fine.

Reply:
You need to import the certificate as an Authority one:
![image](https://user-images.githubusercontent.com/2180032/108196466-6fdfef80-7119-11eb-8619-d6b28857e9ba.png)

Reply:
ok, it works.
so we can remove  the p12 generation, but add a few words in the doc ("install pem for dummies" ?)